src/overlays/actors/ovl_En_Fhg_Fire/z_en_fhg_fire.c

```diff
@@ -462,71 +462,70 @@ void EnFhgFire_Update(Actor* thisx, GlobalContext* globalCtx) {
     this->actionFunc(this, globalCtx);
 }
 
-// Unsolved regalloc
-#ifdef NON_EQUIVALENT
 void EnFhgFire_Draw(Actor* thisx, GlobalContext* globalCtx) {
     EnFhgFire* this = THIS;
-    GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
-    Gfx* dispRefs[4];
+    s32 pad;
 
-    Graph_OpenDisps(dispRefs, gfxCtx, "../z_en_fhg_fire.c", 1723);
-    if (thisx->params == 0x24) {
-        func_80093D84(globalCtx->state.gfxCtx);
-        gDPSetPrimColor(gfxCtx->polyXlu.p++, 0, 0, 255, 255, 255, (s32)this->unk_160 & 0xFF);
-        gDPSetEnvColor(gfxCtx->polyXlu.p++, 165, 255, 75, 0);
-        gDPPipeSync(gfxCtx->polyXlu.p++);
-        gSPMatrix(gfxCtx->polyXlu.p++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_fhg_fire.c", 1745),
-                  G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-        gSPDisplayList(gfxCtx->polyXlu.p++, SEGMENTED_TO_VIRTUAL(D_0600FCF8));
-    } else if ((thisx->params == 0x26) || (thisx->params == 0x32)) {
-        osSyncPrintf("yari hikari draw 1\n");
-        func_800D1FD4(&globalCtx->mf_11DA0);
-        func_80093D84(globalCtx->state.gfxCtx);
-
-        gDPSetPrimColor(gfxCtx->polyXlu.p++, 0, 0, 0xFF, 0xFF, 0xFF, (s32)this->unk_160 & 0xFF);
-
-        if (this->fireMode > 0) {
-            gDPSetEnvColor(gfxCtx->polyXlu.p++, 0, 255, 255, 0);
-        } else {
+    {
+        GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
```

Comment:
Is the macro block really needed here?

Reply:
@Kelebek1

Reply:
based on maides other prs i think its something hes just doing everywhere

Reply:
took it out